akka-docs/rst/scala/stream/migration-guide-2.0-2.4-scala.rst

```diff
@@ -79,6 +79,17 @@ In Akka 2.4.x this is formulated like so:
 
 .. includecode:: ../code/docs/stream/MigrationsScala.scala#expand-state
 
+Changes in Akka HTTP
+====================
+
+Routing settings parameter name
+-------------------------------
+
+``RoutingSettings`` were previously the only setting available on ``RequestContext``,
+and were accessible via ``settings``. We now made it possible to configure the parsers
+settings as well, so ``RoutingSettings`` is now ``routingSettings`` and ``ParserSetttings`` is
+now accessible via ``parserSettings``.
```

Comment:
Java too, same text and we can add "all settings are convieniently modifiable with prepared `with...(...)` methods now"

-- 
Cheers,
Konrad 'ktoso’ Malawski
Akka @ Typesafe

On 25 January 2016 at 23:50:01, Roland Kuhn (notifications@github.com) wrote:

In akka-docs/rst/scala/stream/migration-guide-2.0-2.4-scala.rst:

> @@ -79,6 +79,17 @@ In Akka 2.4.x this is formulated like so:
>   
>  .. includecode:: ../code/docs/stream/MigrationsScala.scala#expand-state
>   
> +Changes in Akka HTTP
> +====================
> +
> +Routing settings parameter name
> +-------------------------------
> +
> +``RoutingSettings`` were previously the only setting available on ``RequestContext``,
> +and were accessible via ``settings``. We now made it possible to configure the parsers
> +settings as well, so ``RoutingSettings`` is now ``routingSettings`` and ``ParserSetttings`` is
> +now accessible via ``parserSettings``.
is this Scala-only or should there be a Java migration guide entry as well?

—
Reply to this email directly or view it on GitHub.